src/todo/todo.lua

```diff
@@ -143,18 +143,26 @@ function todo.update_current_task_label(player)
 
     -- we may update the button label
     todo.log("updating button label")
-    for _, task in ipairs(global.todo.open) do
+    local count
+    for i, task in ipairs(global.todo.open) do
         if task.assignee == player.name then
             todo.log(serpent.block(task))
             todo.get_maximize_button(player).caption =
-                {"todo.todo_maximize_button_caption",
+                {"",
                  {"todo.todo_list"},
+                 ": ",
                  string.match(task.task, "[^\r\n]+")}
             return
         end
+        count = i
```

Comment:
I think you should count only tasks with no assignee here.
`if (task.assignee == nil) then
  count = count + 1
end`

Or you should change the display from "Tasks available." to "Tasks to be done."

Reply:
hmmm... fair point. #66 :)